packages/assets-controllers/src/NftController.ts

```diff
@@ -261,10 +261,7 @@ export class NftController extends BaseController<NftConfig, NftState> {
   private updateNestedNftState(
     newCollection: Nft[] | NftContract[],
     baseStateKey: 'allNfts' | 'allNftContracts',
-    { userAddress, chainId } = {
-      userAddress: this.config.selectedAddress,
-      chainId: this.config.chainId,
-    },
+    { userAddress, chainId }: { userAddress: string; chainId: Hex },
```

Comment:
Should we define this interface somewhere in here? Its used a lot:
```
type OptionsBagOrWhatever = { userAddress: string; chainId: Hex };
```

Reply:
Maybe, though there are variations. Some are `{ userAddress: string; chainId: Hex };` some are { userAddress: string; networkClientId: NetworkClientId }; some are just `{ networkClientId: NetworkClientId };` and some have those plus other entirely different properties.